app/controllers/slack_controller.rb

```diff
@@ -13,19 +13,19 @@ def give_kudos
     end
   end
 
-  def register
+  def team_auth_callback
```

Comment:
This does not seem related to "Slack emoji support", try to keep you PR's small and focussed, that makes reviewing and merging easier.

Reply:
It relates because the flow for connecting users has changed in order to facilitate the editing of messages.

Before a registration token was used to connect Kudo-O-Matic accounts to Slack accounts but because the editing of Slack messages requires a Slack access token from the author of the message. In the gif you can see the 'This message is a Kudo-O-Matic' text being appended to the original Slack message. 

Therefore the auth flow needed to be redesigned.